Sources/SymbolKit/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+#[[
+This source file is part of the DocC open source project
+
+Copyright (c) 2024 Apple Inc. and the DocC project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
```

Comment:
Same as https://github.com/swiftlang/swift-docc/pull/818#discussion_r1846440444

Is there a reason why this isn't the same text as the license header in the other files?

>This source file is part of the Swift.org open source project
>
>Copyright (c) 2024 Apple Inc. and the Swift project authors
>Licensed under Apache License v2.0 with Runtime Library Exception
>
>See https://swift.org/LICENSE.txt for license information
>See https://swift.org/CONTRIBUTORS.txt for Swift project authors